client/pages/dashboard/dashboard.header.controller.js

```diff
@@ -1,22 +1,17 @@
-angular.module('probedock.dashboardPage').controller('DashboardHeaderCtrl', function(orgEditModal, $scope, $state, $stateParams) {
+angular.module('probedock.dashboardPage').controller('DashboardHeaderCtrl', function(orgEditModal, $scope, $state, states) {
 
-  var modal;
-  $scope.currentState = $state.current.name;
+  states.onStateChangeSuccess($scope, 'org.dashboard.default.edit', function(state, params, resolves) {
 
-  $scope.$on('$stateChangeSuccess', function(event, toState) {
+    var modal = orgEditModal.open($scope, {
+      organizationName: resolves.routeOrgName
```

Comment:
Question: When do we need to inject routeOrgName and when do we use it from resolves ?